website/versioned_docs/version-0.78/_integration-with-existing-apps-kotlin.md

```diff
@@ -424,7 +424,7 @@ class MyReactActivity : ReactActivity() {
 </TabItem>
 </Tabs>
 
-As usual, here the [MainActivity.kt Community template file as reference](https://github.com/react-native-community/template/blob/0.77-stable/template/android/app/src/main/java/com/helloworld/MainApplication.kt)
+As usual, here the [MainActivity.kt Community template file as reference](https://github.com/react-native-community/template/blob/0.77-stable/template/android/app/src/main/java/com/helloworld/MainActivity.kt)
```

Comment:
Thanks for catching that! 👍 The link also should point to the `0.78-stable` branch, since you are editing versioned file for 0.78:

```suggestion
As usual, here the [MainActivity.kt Community template file as reference](https://github.com/react-native-community/template/blob/0.78-stable/template/android/app/src/main/java/com/helloworld/MainActivity.kt)
```